api/templates/twiml_error.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Nit: this template is currently unused - response is constructed in code - so this file should be deleted

Reply:
Hrm ... I think I really wanted to make `TemplateTwiMLRenderer.render` use this template, but IIRC I ran into a snag getting that done and probably forgot to finish it. I updated `TemplateTwiMLRenderer.render` to use it now.